### ISSUE TITLE
[UWP] Javascript Alert not showing up on UWP when SetIsJavaScriptAlertEnabled is set to true

### ISSUE THREAD

**Issue by PureWeen** [xamarin/Xamarin.Forms#2002]
### Description
This doesn't appear to working on UWP
https://github.com/xamarin/Xamarin.Forms/pull/1785

It might be that I'm triggering the alert from a global call to opposed to clicking a button.
```
<body>

    If you didn't see an alert you're in trouble

    <script>
        alert("I am a super awesome message");
    </script>
</body>
```

The alert shows up for Android/iOS

### Steps to Reproduce

[UwpJavaScript.zip](https://github.com/xamarin/Xamarin.Forms/files/1773181/UwpJavaScript.zip)

### Expected Behavior
The javascript alert should show for UWP

### Actual Behavior
It doesn't not show for UWP 

### Basic Information

- Version with issue: Xamarin Forms 2.6.0.233087-nightly
- IDE:
- Platform Target Frameworks: 
  - UWP:  10.0.16299.0
- Nuget Packages:
Xamarin Forms 2.6.0.233087-nightly

**Comment by jassmith** [comment]
Attention @alanag13

**Comment by alanag13** [comment]
I admittedly did not test this for this situation (i did everything in response to user input). Thanks for the ping, I'll definitely take a look.

**Comment by jassmith** [comment]
you know our motto, "No bug too small to assign to whatever schmoe touched it last"

**Comment by alanag13** [comment]
@PureWeen @jassmith so I took a look here and it looks like this is might be a limitation of UWP's webview. The message box will only show if alert() is called after the page is completely loaded.

 If you look at my change in  #1785, all I do is reassign the alert function to something that UWP can use. At first I thought maybe I was just doing this assignment too late during the page's lifecycle, so
 I tried doing the same thing during `WebView.ContentLoading` and `WebView.OnDOMContentLoaded` but neither worked. It seems that in all cases the reassignment script will run *after* all scripts in the DOM have already run (this is why responding to a button click works).

There is, however, a workaround for users:
```
<body>

    If you didn't see an alert you're in trouble

    <script>
setTimeout(function(){ alert('foo'); }, 0)
    </script>
</body>
```
using setTimeout (even for 0 ms) forces the browser to defer the script and continue loading the page, so this causes my reassignment code to run first, which makes everything behave as expected.
